### PR TITLE
Add tests for commands

### PR DESCRIPTION
**Pull request by haoyusimon**

### CODE REVIEW

File: src/test/java/seedu/duke/logic/command/EditCommandTest.java [hunk 1]
```diff
@@ -36,10 +40,136 @@ public void execute_oneItemAlreadyInList_editsNormally() throws Exception {
...
+    }
+
+
+
     @Test
     public void execute_emptyList_throwsItemNotExistException() {
-        testCommand = new EditCommand("test", "1", "selling price", "20");
+        testCommand = new EditCommand("test", "1",
+                "selling price", "20");
+        assertThrows(ItemNotExistCommandException.class, () -> testCommand.execute());
```
Comment (t-l-xin) [inline src/test/java/seedu/duke/logic/command/EditCommandTest.java:77]: maybe change the function exception name to the same one that is thrown

File: src/test/java/seedu/duke/logic/command/ReportCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    private Command testCommand;
+
+    @BeforeEach
+    public void setUp() throws IllegalArgumentModelException, DuplicateShelfModelException {
+        ShelfList.getShelfList().resetShelfList();
+        soldItems = new Shelf("soldItems");
+    }
+
+    @Test
+    public void execute_invalidType_throws() {
```
Comment (t-l-xin) [inline src/test/java/seedu/duke/logic/command/ReportCommandTest.java:34]: complete the throw exception in test function naming

File: src/test/java/seedu/duke/logic/command/SellCommandTest.java [hunk 3]
```diff
@@ -48,13 +44,43 @@ public void execute_itemWithInputIdExists_addsNormally() throws CommandException
...
+     */
 
     @Test
-    public void execute_itemWithInputIdDoesNotExist_throwsItemNotExistException()
-            throws CommandException, ShelfNotExistModelException,
-            IllegalArgumentModelException {
+    public void execute_itemWithInputIdDoesNotExist_throwsItemNotExistException() {
         itemID = "XXXXXXXX";
         testCommand = new SellCommand(itemID);
         assertThrows(ItemNotExistCommandException.class, () -> testCommand.execute());
```
Comment (t-l-xin) [inline src/test/java/seedu/duke/logic/command/SellCommandTest.java:53]: rename the function name exception thrown to the same one that is thrown when tested

**Review by t-l-xin — CHANGES_REQUESTED** [review]
Minor changes to function naming

**Review by yuejunfeng0909 — APPROVED** [review]

File: src/test/java/seedu/duke/logic/command/sales/SalesManagerTest.java [hunk 4]
```diff
@@ -1,4 +1,77 @@
...
+        assertThrows(IllegalArgumentCommandException.class,
+                () -> SalesManager.getSalesManager().filterSoldItems("2021-1", ""));
+    }
+
+    @Test
+    public void filterSoldItems_iLLegalArgumentsAndWithinAPeriod_throwsIllegalArgumentException() {
+        assertThrows(IllegalArgumentCommandException.class,
+                () -> SalesManager.getSalesManager().filterSoldItems("2021-1", "2021-2"));
+    }
+
```
Comment (t-l-xin) [inline src/test/java/seedu/duke/logic/command/sales/SalesManagerTest.java:87]: change the test function exception thrown name

**Review by t-l-xin — CHANGES_REQUESTED** [review]
minor change to test function name

**Review by zh1huang — APPROVED** [review]

**Review by haoyusimon — COMMENTED** [review]
Method names have been changed.

**Review by t-l-xin — APPROVED** [review]
Looks good to merge

**Pull request merged**